src/libs/Localize/index.ts

```diff
@@ -72,6 +76,57 @@ const translationCache = new Map<ValueOf<typeof CONST.LOCALES>, Map<TranslationP
     }, [] as Array<[ValueOf<typeof CONST.LOCALES>, Map<TranslationPaths, string>]>),
 );
 
+/**
+ * Helper function to handle pluralized translations.
+ *
+ * This function takes the result of a translated phrase function,
+ * along with the language, phrase key, and count, and returns the
+ * appropriate pluralized form of the translation.
+ *
+ * It uses the Intl.PluralRules API to determine the correct plural form
+ * based on the count and the language's pluralization rules.
+ *
+ * If the specific plural form is not found, it falls back to the 'other' form.
+ *
+ * @param result - The result of the translated phrase function.
+ * @param language - The language of the translation.
+ * @param phraseKey - The key of the phrase being translated.
+ * @param count - The count used to determine the plural form as a string but numerical form (e.g. '5').
+ * @returns The pluralized form of the translation, or null if not found.
+ */
+function getPluralTranslation<TKey extends TranslationPaths>(
+    result: Record<Intl.LDMLPluralRule, Phrase<TKey>>,
+    language: string,
+    phraseKey: TKey,
+    phraseParams?: Record<string, unknown>,
+): string | null {
+    const pluralRules = new Intl.PluralRules(language);
+    const count = phraseParams?.count ?? 1;
```

Comment:
@brandonhenry We can return early here if there is no count prop in the phraseParams object; there is no need to set 1 here.